backend/src/main/webapp/src/pages/ApplicationPage.tsx

```diff
@@ -32,14 +62,57 @@ export const ApplicationPage: React.FunctionComponent = () => {
                     <Thead>
+                        <Toolbar>
+                            <ToolbarContent>
+                                <ToolbarItem>
+                                    <Button variant='primary' type='submit' onClick={ toggle }> Create Event Type </Button>
+                                    <Modal
+                                        variant={ ModalVariant.medium }
+                                        title='Create Event Type'
+                                        isOpen={ isOpen }
+                                        onClose={ toggle }
+                                    ><Form isHorizontal><FormGroup label='Application Id' fieldId='application-id'
+                                            helperText='This is the id of the application'>
+                                            <TextInput
+                                                type='text'
+                                                value= { applicationId }
+                                                id='application-id' /></FormGroup>
+                                        <FormGroup label='Name' fieldId='name' isRequired
+                                            helperText='This is a short name, only composed of a-z 0-9 and - characters.'>
+                                            <TextInput
+                                                type='text'
+                                                onChange={ setName }
+                                                id='name' /></FormGroup>
+                                        <FormGroup label='Display name' fieldId='display-name' isRequired
+                                            helperText='This is the name you want to display on the UI'>
+                                            <TextInput
+                                                type='text'
+                                                onChange={ setDisplayName }
+                                                id='display-name' /></FormGroup>
+                                        <FormGroup label='Description' fieldId='description'
+                                            helperText='Optional short description that appears in the UI to help admin descide how to notify users.'>
+                                            <TextArea
+                                                type='text'
+                                                onChange={ setDescription }
+                                                id='description' /></FormGroup>
+                                        <ActionGroup>
+                                            <Button variant='primary' type='submit'
+                                                onClick= { onSubmit }>Submit</Button>
+                                            <Button variant='link' onClick={ toggle }>Cancel</Button>
+                                        </ActionGroup>
+                                        </Form><></>
```

Comment:
We could probably put this form in a separate file to reuse it when we want to edit.